src/test/resources/emails/instructorCourseRegisterEmail.html

```diff
@@ -5,6 +5,10 @@
   with the Google ID: <strong>user.googleid</strong>.
 </p>
 
+<p>
+  You can use this <a href="http://localhost:8080">URL</a> to login to TEAMMATES.
```

Comment:
I wonder why the URL is the localhost URL. As we will also run the test against staging server, in that case, it will fail? Can you double check this?

Reply:
I checked that one, earlier I was using `${app.url}`. But since in dev server the `${app.url}` is pointing to `Config.APP_FRONTENDDEV_URL` so can't use that. do you have any suggestions?

Reply:
WIP